addons/website_event/controllers/main.py

```diff
@@ -302,9 +302,13 @@ def registration_confirm(self, event, **post):
             Attendees += Attendees.sudo().create(
                 Attendees._prepare_attendee_values(registration))
 
+        return request.redirect('/event/registration/%s/%s/success' % (Attendees.sudo().id, event.id))
+
+    @http.route(['/event/registration/<model("event.registration"):registration>/<model("event.event"):event>/success'], type='http', auth="public", website=True)
```

Comment:
For best practice can you change to `/event/<model("event.event"):event>/registration/<model("event.registration"):registration>/success` ?

Reply:
Changes Done